lib/Mojo/UserAgent.pm

```diff
@@ -403,27 +403,23 @@ Mojo::UserAgent - Non-blocking I/O HTTP and WebSocket user agent
     ->result->save_to('/home/sri/mojo.tar.gz');
 
   # Non-blocking request
-  $ua->get('mojolicious.org' => sub {
-    my ($ua, $tx) = @_;
+  $ua->get('mojolicious.org' => sub ($ua, $tx) {
     say $tx->result->dom->at('title')->text;
   });
   Mojo::IOLoop->start unless Mojo::IOLoop->is_running;
 
   # Concurrent non-blocking requests (synchronized with promises)
   my $mojo = $ua->get_p('mojolicious.org');
   my $cpan = $ua->get_p('cpan.org');
-  Mojo::Promise->all($mojo, $cpan)->then(sub {
-    my ($mojo, $cpan) = @_;
+  Mojo::Promise->all($mojo, $cpan)->then(sub ($mojo, $cpan) {
```

Comment:
For clarity here I'd consider renaming one set of variables, to demonstrate that they're different, Maybe

```perl
my $mojo_p = $ua->get_p('mojolicious.org');
```

and then non-`_p` suffixes on the results of them. Otherwise the reader may not notice the different variables.

Reply:
You could have said the same about the version before.

Reply:
We don't really use `*_p` for variables, probably best to write out `$mojo_promise`.